docs/devel/release-process.rst

```diff
@@ -27,43 +27,35 @@ Process
 
 The process for cutting a minor release is as follows:
 
-#. Ensure upgrading document exists in docs/admin/upgrading
+#. Ensure upgrading document exists in docs/tasks/upgrading
 
+#. Ensure all strings of versions have been updated:
+
+   * deploy/charts/cert-manager/README.md
+   * docs/getting-started/install/kubernetes.rst
+   * docs/getting-started/install/openshift.rst
+   * docs/getting-started/webhook.rst
+   * docs/tutorials/acme/quick-start/index.rst
 #. Create a new release branch (e.g. ```release-0.5```)
 
 #. Push it to the ```jetstack/cert-manager`` repository
 
 #. Create a pull-request updating the Helm chart version and merge it:
```

Comment:
This step isn't required now as we have `#. Ensure all strings of versions have been updated:` above